CHANGELOG.md

```diff
@@ -3,6 +3,12 @@ aws-parallelcluster-node CHANGELOG
 
 This file is used to list changes made in each version of the aws-parallelcluster-node package.
 
+3.14.0
+------
+
+**CHANGES**
+- Support prioritized and capacity-optimized-prioritized Allocation Strategy
```

Comment:
The change log entry should be more focus on the user experience. Could even simply be "Support new allocation strategies `prioritized` and `capacity-optimized-prioritized` to take into account the subnet prioritization on scale out."